src/thinblock.cpp

```diff
@@ -1315,10 +1315,9 @@ void CThinBlockData::ClearThinBlockData(CNode *pnode, uint256 hash)
 uint64_t CThinBlockData::AddThinBlockBytes(uint64_t bytes, CNode *pfrom)
 {
     pfrom->nLocalThinBlockBytes += bytes;
+    nThinBlockBytes.fetch_add(bytes);
 
```

Comment:
BTW, I think that it would be more efficient to do:
uint64_t ret = nThinBlockBytes.fetch_add(bytes);
return ret+bytes;
because .load() is going to be another barrier operation.